### ISSUE TITLE
radio group could not click randomly

### ISSUE THREAD

**Issue by JackySmith** [fyne-io/fyne#2904]
My fyne version is v1.4.0

I make a some container append to a vbox , every container has a radio group, and the last is a button ,which for to the  next step 

the app run ok, but sometimes  the radio could not clicked on. this will appear about 1 time every 10 time the app run.

and more strangly  When it happened, if I still click on  the button to next step, It will suddenly appear it was clicked .

So it can be clicked but just did not  let me see it's was clicked

because  it was not appeared everytime,  so I can not find why .

I am not similar with the new version , So I can not compare in the newest version.

Did you meet the same circumstance before, Please help me !

**Comment by Jacalz** [comment]
We do not maintain support for the older v1.4.x release series any more. Please consider updating to the relatest release, which is v2.1.4 (https://github.com/fyne-io/fyne/releases/tag/v2.1.4) at the time of writing this. Also, please use the issue templates when opening bug reports. They help immensely with our effort to try and replicate as well as fix reported issues. Thanks for understanding.

**Comment by JackySmith** [comment]
Because I am not familiar with new version . And the whole project is so big that I can not rebuild the project.
I know it's hard to get a answer . Can you give me some tip if that happened before?

**Comment by Jacalz** [comment]
Sorry, I don’t think I can help you. There is a very big chance that your issue has already been solved in one of the latest releases. 

You must remember that v1.4.0 is one and a half years old at this point. I would definitely suggest updating to a new release. Documentation for doing so can be found at https://developer.fyne.io/api/v2.0/upgrading.html for 2.0 and https://developer.fyne.io/api/v2.1/upgrading.html for 2.1. There should generally not be too much that has chanced apart from the import path. We worked on making breakage as minimal as we could.
